daphne_worker_test/tests/test_runner.rs

```diff
@@ -354,21 +388,25 @@ impl TestRunner {
         &self,
         client: &reqwest::Client,
         agg_info: &InternalAggregateInfo,
-    ) -> DapLeaderProcessTelemetry {
+    ) -> Result<DapLeaderProcessTelemetry, DapAbort> {
```

Comment:
Let's revert this change, since we no longer need and it's causing lots of clutter in this PR.

Reply:
Sounds good, I've done this.